lib/python/treadmill_aws/bootstrap/node/aws.linux/init1/registry.yml

```diff
@@ -0,0 +1,43 @@
+command: |
+  exec /usr/bin/registry serve etc/config.yml
+
+data_dir:
+- path: etc/config.yml
+  content: |
+    # Docker Registry Configuration
+    version: 0.1
+
+    http:
+      addr: 192.168.254.254:5000
```

Comment:
Yes

Reply:
The cell config will need to be updated afterwards to this

```
data:
  docker_registries: ['192.168.254.254:5000']
```

(instead of the couple of ec2 instances we have in there now)